pkg/controller/controller_broker.go

```diff
@@ -547,40 +548,40 @@ func (c *controller) reconcileClusterServiceClassFromClusterServiceBrokerCatalog
 	toUpdate.Spec.Requires = serviceClass.Spec.Requires
 
 	if _, err := c.serviceCatalogClient.ClusterServiceClasses().Update(toUpdate); err != nil {
-		glog.Errorf("Error updating ClusterServiceClass %q from ClusterServiceBroker %q: %v", serviceClass.Spec.ExternalName, broker.Name, err)
+		glog.Errorf("ClusterServiceBroker %q: Error updating ClusterServiceClass %q: %v", broker.Name, serviceClass.Spec.ExternalName, err)
 		return err
 	}
 
 	return nil
 }
 
-// reconcileServicePlan reconciles a ServicePlan after the
-// ClusterServiceClass's catalog has been re-listed.
-func (c *controller) reconcileServicePlan(broker *v1alpha1.ClusterServiceBroker, servicePlan *v1alpha1.ClusterServicePlan) error {
+// reconcileServicePlanFromClusterServiceBrokerCatalog reconciles a
+// ServicePlan after the ServiceClass's catalog has been re-listed.
```

Comment:
nit, maybe use ClusterServicePlan here and in the name.

Reply:
crap, that's a hold-over.